camus-sweeper/src/main/java/com/linkedin/camus/sweeper/CamusHourlySweeperPlanner.java

```diff
@@ -0,0 +1,133 @@
+package com.linkedin.camus.sweeper;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.log4j.Logger;
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormatter;
+
+import com.linkedin.camus.sweeper.utils.DateUtils;
+
+
+public class CamusHourlySweeperPlanner extends CamusSweeperPlanner {
+  private static final String CAMUS_HOURLY_SWEEPER_MAX_HOURS_AGO = "camus.hourly.sweeper.max.hours.ago";
+  private static final String DEFAULT_CAMUS_HOURLY_SWEEPER_MAX_HOURS_AGO = "1";
+  private static final String CAMUS_HOURLY_SWEEPER_MIN_HOURS_AGO = "camus.hourly.sweeper.min.hours.ago";
+  private static final String DEFAULT_CAMUS_HOURLY_SWEEPER_MIN_HOURS_AGO = "1";
+
+  private static final Logger LOG = Logger.getLogger(CamusHourlySweeperPlanner.class);
+
+  private DateTimeFormatter hourFormatter;
```

Comment:
They are not assigned in constructor, but in method "setPropertiesLogger()". This is the how the old code is written and I'm just overriding it and changing dayFormatter to hourFormatter.